Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/dao/impl/TrustBundleDaoImpl.java

```diff
@@ -323,7 +328,7 @@ public void updateLastUpdateError(long trustBundleId, Calendar attemptTime, Bund
         } catch (ConfigurationStoreException cse) {
             DaoUtils.rollbackTransaction(tx);
             throw cse;
-        } catch (Exception e) {
+        } catch (HibernateException e) {
             DaoUtils.rollbackTransaction(tx);
             throw new ConfigurationStoreException("Failed to update bundle last refresh error.", e);
```

Comment:
There's only one other place (out of very many) in this file we include getLocalizedMessage(); if we want to be consistent, we should remove it instead.  That said, I'd like to leave messages as-is unless there's a compelling reason otherwise.